app/stable_appconfig.go

```diff
@@ -36,6 +37,7 @@ func (app *RegenApp) registerUpgradeHandlers() {
 		storeUpgrades := storetypes.StoreUpgrades{
 			Added: []string{
 				group.ModuleName,
+				icahosttypes.StoreKey,
```

Comment:
We should probably have the host disabled upon upgrade and require a governance proposal to enable it.

Reply:
do you mean to edit the upgrade handler to directly set the host params to false? I don't think we can remove the key in the commented line there

in any case, the host submod will be disabled either way, however disabling it manually in the upgrade handler will at least make it's params queryable post-upgrade. wdyt, edit the handler or leave as is?

Reply:
I just left the comment here because it was the only line changed. I do mean setting the host param to false.

Reply:
Edit the handler. It will be queryable either way as long as the module is wired up. Currently the upgrade will set the host to enabled by default. We should require a governance proposal to enable it before adding messages.

Reply:
ah i missed the default genesis setting it to true, i'll set the params in the upgrade handler 👍🏻